portfolio/src/main/webapp/script.js

```diff
@@ -88,3 +88,31 @@ function clearComments() {
     document.getElementById("history").innerHTML = "";
 }
 
+// Load the Visualization API and the corechart package.
```

Comment:
Can you insert those two statements into a function that loads that data and call it explicitly?